docs/Reference/api.md

```diff
@@ -8,7 +8,7 @@ sidebar_position: 1
 
 Refer to the GoDoc for API documentation and examples.
 
-- [Front end](https://pkg.go.dev/github.com/consensys/gnark@{{content_vars.gnark_version}}/frontend)
-- [Back end](https://pkg.go.dev/github.com/consensys/gnark@{{content_vars.gnark_version}}/backend)
+- [Front end](https://pkg.go.dev/github.com/consensys/gnark/frontend)
```

Comment:
keeping the "@v0.7.1 or @v0.8.0" tag is actually helpful when the docs can display doc version that point to an older version. Though it seems now there isn't a drop down menu any more to pick older versions? (that's probably fine by us)

Reply:
It's possible to support versions in docusaurus as well but if it's not mandatory it can reduce the complexity of the docs without it. If we want to bring in the versions do we have a branch or somewhere to find older version docs?

Reply:
less complexity is good :)